sys-kernel/coreos-modules/files/commonconfig-4.14

```diff
@@ -931,3 +931,11 @@ CONFIG_MODULE_SIG_KEY="certs/modules.pem"
 # CONFIG_XZ_DEC_ARMTHUMB is not set
 # CONFIG_XZ_DEC_SPARC is not set
 CONFIG_NR_CPUS=512
+CONFIG_ACPI=y
+CONFIG_HW_RANDOM=y
```

Comment:
This unmodularizes HW_RANDOM on amd64.  Is there a reason to do that?

Reply:
Was there any reason to build it as a module on amd64? It seems to me that this gives us consistent behavior across platforms, and saw no harm in making this option builtin.